packages/workflow/src/TelemetryHelpers.ts

```diff
@@ -470,3 +471,18 @@ export function generateNodesGraph(
 
 	return { nodeGraph, nameIndices, webhookNodeNames };
 }
+
+export function extractLastExecutedNodeCredentialData(runData: IRun) {
```

Comment:
It would be clearer to return `null` instead of `{credentialId: null,  creentialType: null }`